tests/backstop_data/engine_scripts/puppet/mosaico/mailing/preview-plain-text.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+module.exports = async (engine, scenario, viewport) => {
+  await require('./editor-test.js') (engine, scenario, viewport);
+  await engine.click('button[ng-click="doPreview(\'text\')"]');
+  await engine.waitFor('.status-start', { hidden: true });
+  // wait for Modal to re adjust
+  await engine.waitFor(300);
```

Comment:
have you tried not using this arbitrary `waitFor`s?

Reply:
Yes, We need to have this as w/o it the screenshots gets clipped from the top.